cwa/olin/src/lib.rs

```diff
@@ -1,54 +1,92 @@
+/// Main classes to perform the low-level operations
+/// described in the Olin specs. In particular the basic
+/// "actions": open, read, write, close, flush on files.
+/// 
+/// Modules:
+/// * sys
+/// * err
+/// * log
+/// * env
+/// * runtime
+/// * startup
+/// * time
+/// * stdio
+/// * random
+///
+/// ```Resource``` class defines a generic class for a resource
+/// with methods bound for FFI access
```

Comment:
With the [std::io::Read](https://doc.rust-lang.org/std/io/trait.Read.html) and the [std::io::Write](https://doc.rust-lang.org/std/io/trait.Write.html) traits implemented.